docs/gadget-devel/parameters.md

```diff
@@ -4,6 +4,8 @@ sidebar_position: 610
 description: 'Gadget Parameters'
 ---
 
+## Parameters from eBPF programs
```

Comment:
Before this section, you could have an introduction explaining that the parameters are generally grouped by operator. The group name is either the operator name (e.g. `ebpf`) or "custom" for parameters that don't belong to an operator.

Without this explanation, it is not clear that the group name cannot be arbitrarily chosen by the gadget author.

```
params:
  ebpf:
    myparam:
```

If the parameter group name is neither "custom" or an operator name, it will be ignored by IG.

Reply:
Alternatively, instead of grouping parameters by operator, could parameters be generic and have a field saying it belong to an operator?
```diff
 type GadgetMetadata struct {
 ...
         // Params exposed by this gadget. It includes params for different operators
-        Params map[string]map[string]params.ParamDesc `yaml:"params,omitempty"`
+        Params map[string]params.ParamDesc `yaml:"params,omitempty"`

 }
```
```diff
 type ParamDesc struct {
 ...
+    // Owner tell the operator that own this parameter. Can be empty if none.
+    Owner string
 }
```